eNMS/models/automation.py

```diff
@@ -1410,7 +1410,12 @@ def enter_remote_device(self, connection, device):
         for (send, expect) in zip(commands[::2], commands[1::2]):
             if not send or not expect:
                 continue
-            self.log("info", f"Sent '{send}', waiting for '{expect}'", device)
+            self.log(
+                "info",
+                f"Sent '{send if send != commands[4] else 'jump on connect password'}'",
+                f", waiting for '{expect}'",
+                device,
+            )

```

Comment:
Did you test it and it worked ?

Reply:
Yes, it works (sample test code, below - without this, you can see the jump password).  However, the trailing `device` argument is not needed.  I will remove it - thank you.

```
commands = ['enter_jump_server_command', 'expect_username_prompt', 'jump_server_username', 'expect_password_prompt', 'jump_password_HIDE_ME', 'final_expect_prompt']
for (send, expect) in zip(commands[::2], commands[1::2]):
   if not send or not expect:
     continue
   self.log("info", f"Sent '{send if send != commands[4] else 'jump on connect password'}'", f", waiting for '{expect}'")
```

Output (the jump password is not shown):
```
Sent 'enter_jump_server_command' , waiting for 'expect_username_prompt'
Sent 'jump_server_username' , waiting for 'expect_password_prompt'
Sent 'jump on connect password' , waiting for 'final_expect_prompt'
```

Reply:
but the string is split in two strings / arguments, and the log function only takes one: https://github.com/eNMS-automation/eNMS/blob/master/eNMS/models/automation.py#L911-L918
so it looks like `f", waiting for '{expect}'",` will be passed as the `device` keyword, and the device as `change_log`
What am I missing here ?

Reply:
Oh my goodness, thank you Antoine.  The trailing comma should be removed!